gcn/notices/core/DateTime.schema.json

```
@@ -10,10 +10,18 @@
       "description": "Time of the trigger [ISO 8601], ex YYYY-MM-DDTHH:MM:SS.ssssssZ"
     },
     "trigger_time_error": {
-      "type": "array",
-      "items": { "type": "number" },
-      "maxItems": 2,
-      "description": "Trigger time uncertainty [s, 1-sigma], with optional asymmetric uncertainty"
+      "anyOf": [
```

```suggestion
      "oneOf": [
```
oneOf is I think the more correct keyword to use here [to validate against exactly one of the options provided](https://json-schema.org/understanding-json-schema/reference/combining#oneOf)